Kickstarter-iOS/Views/Controllers/SettingsNotificationsViewController.swift

```diff
@@ -89,13 +90,40 @@ internal final class SettingsNotificationsViewController: UIViewController {
   }
 
   private func animatePickerView(isHidden: Bool) {
-    UIView.animate(withDuration: 0.25) {
-      self.emailPickerViewTopConstraint.constant = isHidden
-        ? 0 : -SettingsNotificationsViewController.emailPickerViewHeight
-
-      self.view.setNeedsLayout()
-      self.view.layoutIfNeeded()
-    }
+    UIView.animate(
+      withDuration: 0.25,
+      animations: { [weak self] in
+        guard let self = self else { return }
+
+        if !isHidden {
```

Comment:
Should we combine the nested `if`s here into one? `if !isHidden && AppEnvironment.current.isVoiceOverRunning()`, also on line `115`.